src/SelectMulti.vue

```diff
@@ -264,8 +264,10 @@
 				 * emits the most recently selected value
 				 * *generally not necessary*, if state can be handled w/ v-model alone
 				 */
-				this.$emit('select', option)
-				this.selectedOptions = [...this.selectedOptions, option]
+				if (option) {
```

Comment:
This is great, thank you @ikniaziuk!

I'm going to add a follow up to create a spec for this (realistically, this part doesn't require a spec, but I'd like to start getting in the habit of increasing our coverage, so the spec will be more about "does pressing enter on an option work as expected")